handlers/kinetic_request_ce_submission_db_insert_v1/handler/init.rb

```diff
@@ -882,8 +882,24 @@ def update_form_table_columns(args)
       end
       if columns_to_add.empty? == false
         puts "Adding the new columns '#{columns_to_add.join(",")}' to #{form_table_name}" if @enable_debug_logging
+
+        # assign variables to be local (there was an issue using class variables inside the .each)
+        using_oracle = @using_oracle
```

Comment:
Single @ is an instance variable and not a class variable. Also what was the issue with using an instance variable inside the each? I can't imagine what sort of issue this assignment would solve.

Reply:
It's been too long for me to remember the details of the issue, but I know that the behavior did work correctly when @using_oracle was used a line 893.

I can change the comment easily enough, but it takes some time to set up and test changes to the handler.  Are you requesting a change?

Reply:
I think its worth understanding the error, because without that context this code makes zero sense to me.